light-zk.js/src/wallet/user.ts

```diff
@@ -1813,12 +1811,10 @@ export class User {
         }
       }
     };
-    let res = undefined;
-    for (var [, programBalance] of this.balance.programBalances) {
-      res = iterateOverTokenBalance(programBalance.tokenBalances);
+    for (const [, programBalance] of this.balance.programBalances) {
+      let res = iterateOverTokenBalance(programBalance.tokenBalances);
       if (res) return res;
     }
-    res = iterateOverTokenBalance(this.balance.tokenBalances);
-    return res;
+    return iterateOverTokenBalance(this.balance.tokenBalances);
   }
```

Comment:
this comment is not related to this line but the User.init method.
Right now User.init returns a value of type any which is really annoying.
Can you change it to return an object of type user?

Reply:
Fixed.